grails-app/views/finder/_auv_functions_js.gsp

```diff
@@ -311,13 +309,13 @@
                 + "&version=1.0.0&maxfeatures=1&sortby=" + variable + "+a";
 
 
-        var xmlDoc = getXML(url);
+        getXML(url, function(doc) {
+            var xmlDoc = doc;
```

Comment:
Does this function do anything? I would think that the `xmlDoc` variable is only in scope within this function, meaning that the function doesn't really do anything.